WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2302,9 +2304,7 @@ private GutenbergPropsBuilder getGutenbergPropsBuilder() {
                 enableXPosts,
                 isUnsupportedBlockEditorEnabled,
                 unsupportedBlockEditorSwitch,
-                !isFreeWPCom, // Disable audio block until it's usable on free sites via "Insert from URL" capability
-                // Only enable reusable block in WP.com sites until the issue
-                // (https://github.com/wordpress-mobile/gutenberg-mobile/issues/3457) in self-hosted sites is fixed
+                !isFreeWPCom,
```

Comment:
These changes come from this [PR](https://github.com/wordpress-mobile/WordPress-Android/pull/14680), that it was required to merge it here due to conflicts in the Gutenberg reference.